dev_deps.bzl

```diff
@@ -0,0 +1,44 @@
+"dev dependencies needed to develop rules_proto"
```

Comment:
Please use triple quotes for DocString of the module.

Reply:
nit: should that be in buildifier then? I don't think it scales to do formatting nits on every ruleset.